packages/core-components/src/layout/HeaderTabs/HeaderTabs.tsx

```diff
@@ -57,18 +58,42 @@ const useStyles = makeStyles(
   { name: 'BackstageHeaderTabs' },
 );
 
-export type Tab = {
+export type TabProp = {
   id: string;
   label: string;
   tabProps?: TabProps<React.ElementType, { component?: React.ElementType }>;
 };
 
 type HeaderTabsProps = {
-  tabs: Tab[];
+  tabs: TabProp[];
   onChange?: (index: number) => void;
   selectedIndex?: number;
 };
```

Comment:
Let's keep this renaming out of this PR for simplicity - no real value in calling it `TabProps` as there is no exported `Tab ` component from this file.

Reply:
checkout `yarn tsc` to see current issue with renaming